src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -284,13 +297,17 @@ def check_no_duplicates(obj: Any) -> None:
         raise ValueError(f"Value {str(obj)} contains duplicate values")
 
 
-class TC_DeviceBasicComposition(MatterBaseTest, BasicCompositionTests):
+class TC_DeviceBasicComposition(BasicCompositionTests, MatterBaseTest):
     @async_test_body
     async def setup_class(self):
         super().setup_class()
         await self.setup_class_helper()
         self.build_spec_xmls()
 
+    @async_test_body
```

Comment:
Does adding this do anything? Does the super not already get called if you don't override?